src/renderer/component/fileList/view.jsx

```diff
@@ -30,12 +30,13 @@ const sortFunctions = {
 type FileInfo = {
   name: string,
   channelName: ?string,
+  pending?: boolean,
   value?: {
     publisherSignature: {
       certificateId: string,
     }
   },
-  metadata: {
+  metadata?: {
```

Comment:
Is this `metadata?` because the API is inconsistent about returning metadata? In general, I'm in favor of consistent signatures that include `null` or otherwise empty values (e.g. `[]`), rather than omitting fields entirely.

Reply:
That part specifically is for pending publishes. A pending publish won't have the `metadata` object. Maybe the issue is not keeping it the same. I know there was a lot of code to build an actual dummy claim in `lbry.js`, currently `pendingPublishes[0]` is the payload we send to `lbry.publish`

Reply:
Yeah, I'm going to add the dummy claim stuff back inside the pending publish. I ran into a couple issues with it, and I'm sure I will only run into more